workflows/go/sdk/order-processor/go.mod

```diff
@@ -0,0 +1,29 @@
+module dapr_example
+
+go 1.21
+
+toolchain go1.21.6
```

Comment:
Please just remove this line, which is unusual and breaks the test.  Line 3 with `go 1.21` directive is good enough to state desired version.
```suggestion
```

Reply:
I'd argue it's not unusual... on upgrading from <1.21 to any package that declares 1.21 will automatically include the toolchain declaration. I believe the reason it is passing for you is that you likely have 1.21 and the test action is currently using go1.18, there is nothing to suggest the tests would pass if this line is removed.

I included the fix in #986 that should address the root cause and get the test passing if it's picked into the 1.13 branch. I set go to 1.21 as this was erroneous but the toolchain directive should remain at parity with the SDK in my opinion.

Reply:
I've cherry-picked the change and included it in this PR. I believe 🤞 the go validation should now pass.

Reply:
Ok good call @mikeee - thank you.